### PR TITLE
clean up, some links

### PR DESCRIPTION
**Pull request by kkraune**
search def doc should be improved, nexts PRs. parent child doc better now

### CODE REVIEW

File: documentation/parent-child.html [hunk 1]
```diff
@@ -21,84 +23,71 @@
...
+</ul>
 </p>
 
 
 
-<h2 id="multivalue-fields">Multivalue fields</h2>
+<h2 id="multi-value-fields">Multi-value fields</h2>
 <p>
-A document can have fields that are arrays of structs.
+A document can have <a href="search-definitions.html#field">fields</a> that are arrays of structs.
```
Comment (geirst) [inline documentation/parent-child.html]: Consider adding maps of struct as well.

File: documentation/search-definitions.html [hunk 2]
```diff
@@ -82,6 +69,19 @@
 
 
 
+<h2 id="field">field</h2>
```
Comment (geirst) [inline documentation/search-definitions.html]: Change to uppercase f in title?

Comment (kkraune) [reply documentation/search-definitions.html]: I could not make up my mind, as others have lowercase as it is the exact name in the search def. will revisit

File: documentation/parent-child.html (hunk 1 above)
Comment (kkraune) [reply documentation/parent-child.html]: yup - added

**Comment by geirst** [comment]
👍

**Pull request merged**